src/wp-includes/class-wp-query.php

```diff
@@ -5042,9 +5103,14 @@ protected function generate_cache_key( array $args, $sql ) {
 			sort( $args['post_status'] );
 		}
 
-		// Add a default orderby value of date to ensure same cache key generation.
```

Comment:
Another nitpick :) This comment still seems okay imho. No need to delete it, just edit it?